compiler/plugins/target/ROCM/test/external_function_validation.mlir

```diff
@@ -0,0 +1,60 @@
+// RUN: iree-opt --pass-pipeline='builtin.module(iree-hal-transformation-pipeline{start=executable-configurations})' \
```

Comment:
you can use `--iree-hal-transformation-pipeline` (shouldn't have to specify a start/end) once simplifying this to just be the executable - you can change then to a public `stream.executable` - see `compiler/plugins/target/ROCM/test/smoketest.mlir`

Reply:
I used the serialization pipeline instead